From cd3170fe15a9b7effe81841ab7ba58d3b525a588 Mon Sep 17 00:00:00 2001
From: Tomas Novotny <tomas.novotny@tbs-biometrics.com>
Date: Tue, 17 Mar 2015 13:45:29 +0100
Subject: [PATCH] ucl: use host compiler supplied by OE

UCL build failed because CC was redefined by OE to the target's compiler.

Upstream-Status: Pending

Signed-off-by: Tomas Novotny <tomas.novotny@tbs-biometrics.com>
---
 tools/ucl/makefile | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/tools/ucl/makefile b/tools/ucl/makefile
index 38d4472..b8ade98 100755
--- a/tools/ucl/makefile
+++ b/tools/ucl/makefile
@@ -1,7 +1,6 @@
 uclobjs = $(obj)ucl_linux
 UTILOBJS = $(patsubst %.c,$(uclobjs)/%.o,$(wildcard src/*.c))
 
-CC=gcc
 AR=ar
 LD=gcc
 RM=rm -rf 
@@ -17,11 +16,11 @@ all: $(UTILOBJS)
 
 $(uclobjs)/%.o : %.cpp
 	$(MKDIR) -p $(@D)
-	$(CC) -c $(CFLAGS) $< -o $@
+	$(HOSTCC) -c $(CFLAGS) $< -o $@
 	
 $(uclobjs)/%.o : %.c
 	$(MKDIR) -p $(@D)
-	$(CC) -c $(CFLAGS) $< -o $@
+	$(HOSTCC) -c $(CFLAGS) $< -o $@
 
 clean:
 	$(RM) $(uclobjs)
-- 
1.8.3.1

